Source/WebKit/UIProcess/RemoteLayerTree/RemoteAcceleratedEffectStack.h

```diff
@@ -55,21 +55,24 @@ class RemoteAcceleratedEffectStack final : public WebCore::AcceleratedEffectStac
     void clear(PlatformLayer*);
 
 private:
-    explicit RemoteAcceleratedEffectStack(Seconds);
+    explicit RemoteAcceleratedEffectStack(WebCore::FloatRect, Seconds);
 
     WebCore::AcceleratedEffectValues computeValues(MonotonicTime now) const;
 
     enum class LayerProperty : uint8_t {
         None = 1 << 0,
```

Comment:
Odd to have a None value for a bitset. None is just no bits set.

Reply:
Yes, this is useless and I'll remove it when landing.